ext/microprofile/mp-rest-client/pom.xml

```diff
@@ -114,6 +114,8 @@
                             org.glassfish.jersey.restclient.*;version=${project.version}
                         </Export-Package>
                         <Import-Package>
+                            ${cdi.osgi.version},
+                            org.eclipse.microprofile.rest.client.*;version=[1,3),
```

Comment:
```suggestion
                            org.eclipse.microprofile.rest.client.*;version="[1,3)",
```

Reply:
Sounds right thing to do, but it seemed to work anyway in Payara...will fix it though

Reply:
Thanks!